CONTRIBUTING.md

```diff
@@ -31,13 +31,13 @@ and "help wanted" is open to whoever wants to implement it.
 
 ### Write Documentation
 
-samplingsimulatorpy could always use more documentation, whether as part of the
-official samplingsimulatorpy docs, in docstrings, or even on the web in blog posts,
+samplingsimulatorr could always use more documentation, whether as part of the
+official samplingsimulatorr docs, in docstrings, or even on the web in blog posts,
 articles, and such.
 
 ### Submit Feedback
 
-The best way to send feedback is to file an issue at https://github.com/UBC-MDS/samplingsimulatorr/issues.
+The best way to send feedback is to file an issue at [here](https://github.com/UBC-MDS/samplingsimulatorr/issues).
```

Comment:
Suggest deleting "at".
```suggestion
The best way to send feedback is to file an issue [here](https://github.com/UBC-MDS/samplingsimulatorr/issues).
```

Reply:
Thanks for pointing out! I just fixed it. Let me know if it needs additional changes.